### PR TITLE
FIX: Avoid bang methods in Disqus importer.

### PR DESCRIPTION
**Pull request by tgxworld**
Meta: https://meta.discourse.org/t/disqus-importer-fails-undefined-method/36200

### CODE REVIEW

File: script/import_scripts/disqus.rb [hunk 1]
```diff
@@ -151,8 +151,7 @@ def normalize
         @threads.delete(id)
       else
         # Normalize titles
-        t[:title].gsub!(@strip, '') if @strip.present?
-        t[:title].strip!
+        t[:titlet] = [:title].gsub(@strip, '').strip if @strip.present?
```
Comment (ZogStriP) [inline script/import_scripts/disqus.rb]: `titlet`?

Comment (tgxworld) [inline script/import_scripts/disqus.rb]: oops will fix!

**Comment by yyhmsg** [comment]
Thanks @tgxworld for the quick fix.  The importer seems to work now - or at least my dry_run worked, I'll try some more tests first and then let you know if there are any issues with production.

**Comment by tgxworld** [comment]
Merging this in :smile:

**Pull request merged**